src/Notes.js

```diff
@@ -0,0 +1,32 @@
+import ERROR_MESSAGES from './Error/Error.js';
+import { isNumber } from './Util/Regex.js';
+
+// Lotto와 함께 src 루트 폴더에 보관
+class Notes {
+  #notes;
+
+  constructor(paidAmount) {
+    const parsedNotes = Number.parseInt(paidAmount / 1000, 10);
```

Comment:
parseNotes 혹은 paidAmount를 인스턴스 변수로 설정하여 메소드 내부에서 접근할 수 있는 방법도 있는데

로컬 변수로 지정하신 이유가 궁금합니다.
```suggestion
    const parsedNotes = Number.parseInt(paidAmount / 1000, 10);
```

Reply:
일반적인 상태(#notes)와 다르게 '상태에서 파생되는 상태'(parsedNotes)는 필드로 분리하면 상태 관리가 많이 불리해지기 때문입니다.